leapfrog/scripts/benchmark.md

```diff
@@ -1,6 +1,6 @@
 ## Benchmarking
 
-The benchmark script can be used to run the different iterations fo the model via R to get timings. Initially this script was used to compare leapfrog and frogger timings. Since we want to replace leapfrog with frogger, we'll no longer be able to run them separately. This document has some timings from two machines for posterity.
+The benchmark script can be used to run the different iterations fo the model via R to get timings. Initially this script was used to compare leapfrog and leapfrog timings. Since we want to replace leapfrog with leapfrog, we'll no longer be able to run them separately. This document has some timings from two machines for posterity.
```

Comment:
>Since we want to replace leapfrog with leapfrog, we'll no longer be able to run them separately. This document has some timings from two machines for posterity.

This text needs some revision.